lib/themes/dosomething/paraneue_dosomething/templates/fact/node--fact_page.tpl.php

```diff
@@ -81,7 +81,7 @@
             <h4 class="js-footnote-toggle"><?php print t('Sources'); ?></h4>
             <ul class="js-footnote-hidden">
               <?php foreach ($sources as $key => $source): ?>
-                <li><sup><?php print ($key + 1); ?></sup> <?php print $source; ?></li>
+                <li><sup><?php print($key + 1); ?></sup> <?php print $source; ?></li>
```

Comment:
Hmm, this isn't really a comment on the fixer, but it could be worthwhile to standardize on whether we want to use parentheses for language constructs like `print` or `echo`. It's a little weird seeing it both ways on one line.